.eslintrc.js

```diff
@@ -84,7 +84,7 @@ module.exports = {
 
     // TODO
     'react/prop-types': 'off', // For public API
-    'import/prefer-default-export': 'warn',
+    'import/prefer-default-export': 'off',
```

Comment:
Creates too many warning. 
Don't like default as, it breaks refactoring tool and a var can be named x in one import and y in another. Why is that great? And if you decide to rename it Z it won't propagate...

Reply:
In this case, please contribute in https://github.com/mui-org/material-ui/issues/21862 and leave the rule in place until we find a resolution.